common/containers/Tabs/SendTransaction/components/UnitDropdown.jsx

```diff
@@ -1,87 +1,40 @@
 // @flow
 import React from 'react';
+import SimpleDropDown from 'components/ui/SimpleDropdown';
 
-class Option extends React.Component {
-  props: {
-    value: string,
-    active: boolean,
-    onChange: (value: string) => void
-  };
-  render() {
-    const { value, active } = this.props;
-    return (
-      <li>
-        <a className={active ? 'active' : ''} onClick={this.onChange}>
-          {value}
-        </a>
-      </li>
-    );
-  }
-
-  onChange = () => {
-    this.props.onChange(this.props.value);
-  };
-}
+type UnitDropdownProps = {
+  value: string,
+  options: string[],
+  onChange?: (value: string) => void
+};
 
 export default class UnitDropdown extends React.Component {
-  props: {
-    value: string,
-    options: string[],
-    onChange?: (value: string) => void
-  };
+  props: UnitDropdownProps;
+
   state: {
     expanded: boolean
   } = {
     expanded: false
   };
 
   render() {
-    const { value, options, onChange } = this.props;
-    const isReadonly = !onChange;
+    const { value, options } = this.props;
 
     return (
       <div className="input-group-btn">
-        <a
-          style={{ minWidth: 170 }}
+        <SimpleDropDown
           className="btn btn-default dropdown-toggle"
```

Comment:
Not sure how I feel about this, we should probably have a separate component for this. The dropdown component that Bootstrap provides doesn't behave like this, it needs completely different markup than a select list.

Reply:
Or just be able to toggle the styling, if that's all we care about.

Reply:
Yeah agreed, the styling needs to be different. I meant it sort of as a placeholder, but we can remove it altogether.

Alternatively, we could create a `NonSimpleDropDown` that uses the fancy `<ul>`s and pretends to be a select menu while also offering more styling.

Reply:
Maybe `SimpleSelect` and `SimpleDropdown`?